app/controllers/agents/relatives_controller.rb

```diff
@@ -1,6 +1,10 @@
 class Agents::RelativesController < AgentAuthController
+  respond_to :html
```

Comment:
not sure this is necessary

Reply:
don't send modal if it isn't set:
`In order to use respond_with, first you need to declare the formats your controller responds to in the class level`